clarify api access (#300)
neo4j/docs-aura: modules/ROOT/pages/platform/api/authentication.adoc

[NOTE]
====

> In addition to Enterprise users, users with Free and Professional instances can also create API credentials as long as they have entered billing information or are a member of a marketplace tenant.

Reviewer: ```suggestion
Enterprise users and users with Free and Professional instances can create API credentials as long as they have entered billing information or are a member of a marketplace tenant.
```

Just not a fan of "users, users".

Author: While I understand your position, the re-write lacks the clarity that it is just the free and pro instance users that need the billing information, not the enterprise users.

Reviewer: Ah, I didn't get that difference. In this case I would rephrase:

```suggestion
Enteprise users can always create API credentials.
Users of Free and Professional instances can create API credentials as long as they have entered billing information or are a member of a marketplace tenant.
```
